File: code/Java/Mobile/TestApplication/TrackPlatformAPI/app/src/main/java/com/example/yuras/trackplatformapi/ServoCommands.java
package com.example.yuras.trackplatformapi;

public enum ServoCommands {
    SET_HORIZONTAL_ANGLE("\003\001"),
    SET_VERTICAL_ANGLE("\003\002"),
    GET_ANGLES("\003\004");

    private final String value;

    ServoCommands(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: code/cs_api/cs_api/CommandsSpecification/ServoCommands.cs
namespace TrackPlatform.CommandsSpecification
{
    public enum ServoCommands
    {
        set_horizontal_angle = '\x0001', //set horizontal angle of servo: obsolete
        set_vertical_angle = '\x0002', //set vertical angle of servo: obsolete
        set_horiz_vertical_angles = '\x0003', //set horizontal and verticalse angles of servo: obsolete
        get_coodrinates = '\x0004', //get current angels of horizontal and vertical servos: obsolete
        set_angle = '\x05', //set axis angle
        get_angle = '\x06', //get axis angle
    }
}